webpack.config.js

```diff
@@ -10,10 +10,23 @@ module.exports = {
           loader: 'babel-loader', // Allows Babel and Webpack to work together
           options: {
```

Comment:
should remove this options property and use a `babel.config.json` file

Reply:
Good call, updated to be correct